src/music_gui.cpp

```diff
@@ -26,6 +26,7 @@
 #include "string_func.h"
 #include "settings_type.h"
 #include "settings_gui.h"
+#include "tilehighlight_func.h"
```

Comment:
it's strange to require tilehighlite_func when no working with tiles at all

Reply:
I agree -_- this is required to use `SetObjectToPlaceWnd()`. `depot_gui.cpp` uses this function in a similar way, to allow for dragging vehicles to be sold in the depot window.

Maybe the window system's dragging events were only anticipated to be used with tiles? I think it may make sense to move `SetObjectToPlaceWnd()` out of `tilehighlight_func.h`, as it's useful for dragging events inside windows that don't work with tiles at all.